resources/lang/ell.ini

```diff
@@ -72,30 +72,30 @@ flags.natural = "Φυσική"
 ; tools
 ; wand tool
 tool.wand = "Ραβδί"
-tool.wand.lore.1 = "Left click a block to set the position 1 of a selection"
-tool.wand.lore.2 = "Right click a block to set the position 2 of a selection"
-tool.wand.lore.3 = "Use //togglewand to toggle it's functionality"
-tool.wand.disabled = "The wand tool is disabled. Use //togglewand to re-enable it"
-tool.wand.setenabled = "The wand tool is now {%0}!"
+tool.wand.lore.1 = "Κάντε αριστερό κλικ σε ένα block για να θέσετε τη τοποθεσία 1 από μια επιλογή"
+tool.wand.lore.2 = "Κάντε δεξί κλικ σε ένα block για να θέσετε τη τοποθεσία 2 από μια επιλογή"
+tool.wand.lore.3 = "Χρησιμοποιείστε το //togglewand για να αλλάξετε τη λειτουργία του"
+tool.wand.disabled = "Το εργαλείο ραβδί είναι απενεργοποιημένο. Χρησιμοποιείστε το //togglewand "
+tool.wand.setenabled = "Το εργαλείο ραβδί είναι τώρα {%0}!"
 ; debug tool
 tool.debug = "Εργαλείο αποσφαλμάτωσης"
-tool.debug.lore.1 = "Left click a block to get information"
-tool.debug.lore.2 = "like the name and damage values of a block"
-tool.debug.lore.3 = "Use //toggledebug to toggle it's functionality"
-tool.debug.disabled = "The debug tool is disabled. Use //toggledebug to re-enable it"
-tool.debug.setenabled = "The debug tool is now {%0}!"
+tool.debug.lore.1 = "Κάντε αριστερό κλικ στο block για να πάρετε πληροφορίες"
+tool.debug.lore.2 = "σαν το όνομα και το ποσό ζημιάς ενός block"
+tool.debug.lore.3 = "Χρησιμοποιείστε το //toggledebug να αλλάξετε τη λειτουργία του"
+tool.debug.disabled = "Το εργαλείο αποσφαλμάτωσης είναι απενεργοποιημένο. Χρησιμοποιείστε το //toggledebug για να το επανενεργοποιήσετε"
+tool.debug.setenabled = "Το εργαλείο αποσφαλμάτωσης είναι τώρα {%0}!"
 ; flood tool
-ui.flood.title = "Flood menu"
+ui.flood.title = "Μενού πλημμύρας"
 ui.flood.options.limit = "Μέγιστα blocks"
 ui.flood.options.blocks = "Blocks"
```

Comment:
Thinking of the blocks, is it like a "specific" term in greece? Blocks are the breakable Voxels in the game Minecraft, i am unsure if minecraft has a specific word for them in greece. Maybe a fitting translation can be found here: https://mcpedl.com/translations-for-minecraft/?cookie_check=1

Reply:
I've been thinking about it and probably the best localization is "τουβλάκια". I'm pushing a commit on this